clients/python/README.md

```diff
@@ -3,8 +3,8 @@ This page is the reference of the Bacalhau REST API. Project docs are available
 
 This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
 
-- API version: ${PYPI_VERSION}
-- Package version: 0.3.24.dev8
+- API version: ${VERSION}
+- Package version: 
```

Comment:
It will be same as our git version. 

https://github.com/bacalhau-project/bacalhau/blob/main/scripts/get_current_version_for_builds.py

This is the script that gets used.